### ISSUE TITLE
QuickPick buttons do not render

### ISSUE THREAD

**Issue by SamuelBergSTM** [eclipse-theia/theia#13076]
### Bug Description:

QuickPick buttons do not render in Theia. They render as expected in VS Code.

### Steps to Reproduce:

1. Create a QuickPick with a button.
2. Await to be able to keep QuickPick shown.
3. Show the QuickPick.

Example:

```ts
const quickPick = vscode.window.createQuickPick();
quickPick.buttons = [vscode.QuickInputButtons.Back];

await new Promise<void>(resolve => {
    quickPick.onDidAccept(() => {
        resolve();
    });

    quickPick.show();
});
```

### Additional Information

- Operating System: Windows 10 Enterprise
- Theia Version: 1.43.0

QuickPick with Back button in VS Code:

![image](https://github.com/eclipse-theia/theia/assets/93371821/232c576d-f6da-45ff-9da0-10609c48a6d3)

QuickPick with Back button in Theia:

![image](https://github.com/eclipse-theia/theia/assets/93371821/0bb30d53-b2ce-4d9e-a8b7-a8a7285f8b36)

**Comment by tortmayr** [comment]
I cannot reproduce this issue.
Using the example provided the back  button is properly rendered in both 1.43.x and the current master (tested on Windows and Ubuntu).

**Comment by JonasHelming** [comment]
@SamuelBergSTM : In order to reproduce this, could you maybe:
Provide a video or the exact steps (click here, hit these keys, etc.)
Provide a an example extensions (yo code makes this quite painless)
Provide browser and console logs (there might be an error)

**Comment by SamuelBergSTM** [comment]
[theia-13076.zip](https://github.com/eclipse-theia/theia/files/13965166/theia-13076.zip)

@JonasHelming I attached a ZIP file including:

1. `demo.mkv`: A video demonstrating how it is reproduced.
2. `steps.txt`: Steps for creating the extension and reproducing the issue. Also includes versions of relevant software.
3. `logs`: Frontend and backend logs.
4. `theia-13076-0.0.1.vsix`: VSIX for the extension.
5. `theia-13076`: Source files for the extension (excluding `node_modules` and `out` folders).

**Comment by tortmayr** [comment]
@SamuelBergSTM  Thaks for the detailed instructions.
I will try to reproduce the issue and investigate a potential fix.

**Comment by tortmayr** [comment]
@SamuelBergSTM This issue should be fixed with https://github.com/eclipse-theia/theia/pull/13342

**Comment by SamuelBergSTM** [comment]
> @SamuelBergSTM This issue should be fixed with #13342

Perfect. Thanks.
